azure-cs-aks-private-container-registry/Program.cs

```diff
@@ -0,0 +1,216 @@
+﻿// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Pulumi;
+using Pulumi.AzureAD;
+using Pulumi.Azure.ContainerService;
+using Pulumi.Azure.ContainerService.Inputs;
+using Pulumi.Azure.Core;
+using Pulumi.Azure.Network;
+using Pulumi.Azure.Role;
+using Pulumi.Docker;
+using Pulumi.Kubernetes.Types.Inputs.Apps.V1;
+using Pulumi.Kubernetes.Types.Inputs.Core.V1;
+using Pulumi.Kubernetes.Types.Inputs.Meta.V1;
+using Pulumi.Random;
+using Pulumi.Tls;
+using ContainerArgs = Pulumi.Kubernetes.Types.Inputs.Core.V1.ContainerArgs;
+using SecretArgs = Pulumi.Kubernetes.Types.Inputs.Core.V1.SecretArgs;
+using ServiceArgs = Pulumi.Kubernetes.Types.Inputs.Core.V1.ServiceArgs;
+
+class Program {
+	static Task<int> Main() {
+		return Deployment.RunAsync(() => {
+			var resourceGroup = new ResourceGroup("aks-rg");
```

Comment:
Could you please use the standard indentation of 4 spaces everywhere? Thanks!

Reply:
Mysteriously enough, the editorconfig in the samples root caused Rider to apply different settings - removing editorconfig allowed Rider to stick to my system-defaults (which includes 4 spaces everywhere, BSD style braces, etc. Anyway, I will fix.